wsclient/src/main/java/org/hpccsystems/ws/client/antlr/gen/EclRecord.interp

```diff
@@ -0,0 +1,109 @@
+token literal names:
```

Comment:
I think this file should probably not be committed, and falls under my comment about expanding the .gitignore file